demo/src/app/components/alert/alert.component.ts

```diff
@@ -1,12 +1,24 @@
 import {Component} from '@angular/core';
-import {NgbdApiDocs} from '../shared/api-docs';
+import {ExampleBoxComponent, NgbdApiDocs} from '../shared';
+import {AlertBasicComponent, basicHtmlContent, basicTsContent,
+        AlertCloseableComponent, closeableHtmlContent, closeableTsContent} from './demos';
 
 @Component({
   selector: 'ngbd-alert',
   template: `
     <ngbd-api-docs directive="NgbAlert"></ngbd-api-docs>        
+    <ngbd-example-box demoTitle="Basic demo" [htmlSnippet]="basicHtmlContent" [tsSnippet]="basicTsContent">
+      <ngbd-alert-basic></ngbd-alert-basic>
+    </ngbd-example-box>
+    <ngbd-example-box demoTitle="Closeable demo" [htmlSnippet]="closeableHtmlContent" [tsSnippet]="closeableTsContent">
+      <ngbd-alert-closeable></ngbd-alert-closeable>
+    </ngbd-example-box>
   `,
-  directives: [NgbdApiDocs]
+  directives: [AlertBasicComponent, AlertCloseableComponent, NgbdApiDocs, ExampleBoxComponent]
 })
 export class NgbdAlert {
+  basicHtmlContent = basicHtmlContent;
```

Comment:
I really don't like this paradigm as it doesn't scale with multiple demos.  I _think_ this has more to do with how we are doing exports, but we'll still need the names to be uniqe...I think...based on how we've got the wrapper demo constructed.